docs/processes/cldr-icu.md

```diff
@@ -272,30 +273,17 @@ already present in the ICU4C sources. This process uses the `LdmlConverter` in
 `$ICU_DIR/tools/cldr/cldr-to-icu/`; see `$ICU_DIR/tools/cldr/cldr-to-icu/README.md`.
 
 * This process will take several minutes, during most of which there will be no log
-  output (so do not assume nothing is happening). Keep a log so you can investigate
+  output (so do not assume that nothing is happening). Keep a log so you can investigate
   anything that looks suspicious.
-* The conversion tool
-  will automatically run its own "clean" step to delete files it cannot determine to
-  be ones that it would generate, except for pasts listed in `<retain>` elements such as
-  `coll/de__PHONEBOOK.txt`, `coll/de_.txt`, etc.
+* The conversion tool will automatically run its own "clean" step to delete files it
+  cannot determine to be ones that it would generate, except for pasts listed in
+  `<retain>` elements such as `coll/de__PHONEBOOK.txt`, `coll/de_.txt`, etc.
 * Before running the tool to regenerate the data, make any necessary changes to the
   `config.xml` file, such as adding new locales etc.
-* **Temporary note 2025-04-07:** There are some steps mentioned in `$ICU_DIR/tools/cldr/cldr-to-icu/README.md`
-  that were not mentioned in these instructions but seem to be necessary for the next step to
-  work properly, these are:
-    * Build ICU4J:
-    ```
-    cd "$ICU_DIR"
-    mvn clean install -f icu4j -DskipTests -DskipITs
-    ```
-    * Build the conversion tool:
-    ```
-    cd "$ICU_DIR/tools/cldr/cldr-to-icu/"
-    mvn clean package -DskipTests -DskipITs
-    ```
-
-```sh
-cd $ICU_DIR/tools/cldr/cldr-to-icu
+
+```sh
+cd $TOOLS_ROOT/cldr/cldr-to-icu
+mvn clean package -DskipTests -DskipITs
```

Comment:
LGTM.
This works now because line 205 changed from `mvn verify` to `mvn install`

So there is no need to run `mvn install` here anymore.